### DISCUSSION TITLE
Show full title of pinned discussions when hovering over

### DISCUSSION THREAD

**Discussion by JoseDeFreitas** [community/community#6167] [Discussions]
Currently, when you hover over the title of a discussion that is pinned, there's no `Title` attribute in the HTML that shows the full title. I've come across pinned discussions and I can't get to read the entire title (in fact, only a few words can be seen), so I have to open the discussion in a new tab to see what it is about. It would be nice to see at least a pop-up with the entire title provided by the `Title` property of HTML.

**Comment by nul800sebastiaan** [comment]
Was about to suggest the same thing, I have very little context as to what the currently pinned items will actually lead me to.. 

![image](https://user-images.githubusercontent.com/304656/160778981-b260cb7a-2fde-48d1-b5dd-a6d487a35169.png)
